src/renderer/redux/actions/content.js

```diff
@@ -70,6 +70,13 @@ export function doResolveUri(uri) {
 }
 
 export function doFetchFeaturedUris() {
+  return dispatch => {
+    dispatch(doFetchFeaturedUrisNow());
+    setInterval(() => dispatch(doFetchFeaturedUrisNow()), 40000);
+  }
+}
+
+export function doFetchFeaturedUrisNow() {
```

Comment:
Setting an interval like this is absolutely correct, but it would be better if you found the place that originally triggers `doFetchFeaturedUris` and set the interval there instead of creating two actions.